parse_cmd.py

```diff
@@ -8,7 +8,7 @@
 pydict = PyDictionary()
 
 # Set up defined directions, determiners, and prepositions
-directions = {"NORTH", "SOUTH", "EAST", "WEST"}
+directions = {"NORTH", "SOUTH", "EAST", "WEST", "NORTHWEST", "NORTHEAST", "SOUTHEAST", "SOUTHWEST"}
```

Comment:
I am worried if we use NORTHWEST users will type NORTH WEST and then the parser as it stands will grab NORTH as the direction... but I don't know if we have enough time to screw with the parser and make it work at this point.  Just pointing it out.